README.md

```diff
@@ -12,13 +12,26 @@ Spring Boot (https://github.com/spring-projects/spring-boot) makes it easy to cr
 
```

Comment:
Can we have a high level description of the Project? I see in README talks about various technologies used but does tell me technologies used for what purpose. For example, what is the information that is stored in DB and how it will be used in the project.